src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -936,7 +936,10 @@ protected async override Task<SearchResult> SearchForReindexInternalAsync(Search
             do
             {
                 // Get surrogate ID ranges
-                ranges = await GetSurrogateIdRanges(resourceType, queryStartId, globalEndId, searchOptions.MaxItemCount, 50, true, cancellationToken);
+                ranges = await GetSurrogateIdRanges(resourceType, queryStartId, globalEndId, searchOptions.MaxItemCount, 10, true, cancellationToken);
```

Comment:
Why reduce the number of ranges to 10?

Reply:
My thinking behind this is the "happy path" for most reindex operations doesn't encounter empty data ranges. Reducing the ranges to 10 will slightly reduce the CPU consumption used by general reindex operations while also not querying the database for every range.

I'm not solid on these numbers - since this is temporary code I did what felt right vs any perf tested. 

For a standard reindex which looks at 100 records at a time, there would need to be 1000 concurrent records of only history to activate the "do/while" part of the loop w/ additional database queries. 1000 seems like a round number and sorting 10 records seems like a lower CPU lift.

Reply:
I think the term [SWAG](https://en.wikipedia.org/wiki/Scientific_wild-ass_guess) is appropriate.